breguet_range.py

```diff
@@ -0,0 +1,62 @@
+from gpkit.shortcuts import *
+from gpkit import Variable, Model, units
+import numpy.testing as npt
+
+class  B_Range(Model):
+
+	"""Breguet Range Model
+
+	Arguments
+	--------------------
+	TSFC_min 
+		Minimum TSFC value, assumed constant
+	MTOW 
+		Max Take Off Weight
+	W_oew 
+		Operating empty weight
+	LoverD_max
+		Maximum Lift to Drag ratio
+	M_max
+		Maximum Mach number 
+
+	"""
+	def setup(self, TSFC_min=0.307, MTOW=10000, W_oew=7000, LoverD_max=15, M_max=0.78):
+		#store attributes for later external use
+		self.TSFC_min, self.MTOW, self.W_oew, self.LoverD_max, self.M_max = TSFC_min, MTOW, W_oew, LoverD_max, M_max
+		TSFC_min = Var('TSFC_{min}', TSFC_min, "lb/lbf/hr")
+		MTOW = Var('MTOW', MTOW, "lbf")
+		W_oew = Var('W_{oew}', W_oew, "lbf")
+		LoverD_max = Var('LoverD_{max}', LoverD_max)
+		M_max = Var('M_max', M_max)
+	
+		#Constants
+		g = Var('g', 9.81, "m/s^2","gravity")
+		a0 = Var('a0', 340.29, "m/s", "speed of sound at sea level")
```

Comment:
@bqpd , re: comment above, note these default values are somehow not expected to be modified as often by users...